docs: add remarks about external sources (#3452)
faker-js/faker: CONTRIBUTING.md

</tr>
</table>

> > Further JSDoc tags are in use for specific cases.
> 
> - `@internal` - If the method is not intended to be used by the end user, e.g. a helper function
> - `@remarks` - If the method depends on external data not in control of Faker, e.g. a third-party image provider
> 

Reviewer: ```suggestion
> Other JSDoc tags are in use for specific cases.

- `@internal` - If the method is not intended to be used by the end user, e.g. a helper function.
- `@remarks` - If the method depends on external data not being controlled by Faker, e.g. a third-party image provider.

```
